.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
you would usually want your settings.json file to be included in .gitignore file